### ISSUE TITLE
Tap gestures not working while using DragAndDrop

### ISSUE THREAD

**Issue by NickeManarin** [roubachof/Sharpnado.Presentation.Forms#61]
**Platform (please complete the following information):**
 - OS: Android
 - Device: AndroidEmulator
 - Sdk vervion: Android SDK 28 (Pie)
 - Xamarin.Forms: 4.0.0.482894

**Describe the bug**
I'm using a `HorizontalListView` with `DraggableViewCell` as the item template. 
Tap events are not triggered on the `ViewCell` (Both the event `Tapped` as the attached command `TapCommandEffect.Tap` and any custom command that I created by extending the `DraggableViewCell` class).
The `TapCommandEffect.Tap` attached command works normally while being used on a `ContentView` inside the `ViewCell`.

**To Reproduce**
```
<r:HorizontalListView x:Name="MainListView" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" ColumnCount="2"
                      CollectionPadding="16" ItemSpacing="8" EnableDragAndDrop="True" ItemHeight="120"
                      ItemsSource="{Binding Items}" ListLayout="Grid">
     <r:HorizontalListView.ItemTemplate>
        <DataTemplate>
            <r:DraggableViewCell x:Name="DraggableViewCell"
                             e:ViewEffect.TouchFeedbackColor="Coral" 
                             e:TapCommandEffect.Tap="{Binding BindingContext.SelectPhotoCommand, Source={x:Reference ThisView}}"
                             e:TapCommandEffect.TapParameter="{Binding .}">
                <ContentView>
                    <Grid Margin="5" BackgroundColor="CadetBlue">
                    </Grid>
                </ContentView>
            </r:DraggableViewCell>
        </DataTemplate>
    </r:HorizontalListView.ItemTemplate>
</r:HorizontalListView>
```

I'm using a ObservableCollection<Object> as my backing list.

**Comment by roubachof** [comment]
A ```TapCommand``` property will be added in the next version for dealing with drag'n'drop scenarios

**Comment by roubachof** [comment]
Fixed in v1.2.0
